hoff.cabal

```diff
@@ -20,6 +20,7 @@ library
   hs-source-dirs:  src
   exposed-modules: Configuration
                  , EventLoop
+                 , Format
```

Comment:
The "library" is only intended to be used in the app binary and in the tests. It would be nice to namespace them still, but as far as I am aware, the file structure on disk has to match the module structure, and I would like to avoid these java-esque situations where you have to open/tab-complete ten boilerplate directories before you get to the source code.

Do you know of a way to prefix the modules with `Hoff.` while keeping the source files directly under `src`?

Reply:
> ten boilerplate directories

Well, it would only be one, but 

> Do you know of a way to prefix the modules with Hoff

No, I think Haskell modules have to correspond to their path on disk.

Reply:
Okay, I will keep it just in `src` for now then.